src/main/java/org/folio/circulation/domain/policy/RollingRenewalDueDateStrategy.java

```diff
@@ -16,7 +16,7 @@
   private static final String RENEW_FROM_DUE_DATE = "CURRENT_DUE_DATE";
 
   private static final String NO_APPLICABLE_DUE_DATE_LIMIT_SCHEDULE_MESSAGE =
-    "renewal date falls outside of date ranges in rolling loan policy";
+    "renewal date falls outside of date ranges in the loan policy";
```

Comment:
Hi @dmytrotur i guess the main reason that now in one 'fixed' case we use rolling strategy. See comment https://issues.folio.org/browse/CIRC-200?focusedCommentId=59100&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-59100

Reply:
I see, the strategy can be both rolling and fixed in this case. The problem is that these error messages are used on UI to identify a particular error and do some actions based on the result:
https://github.com/folio-org/ui-users/blob/734e406fc427ebdc55894585732f447bff8d803a/src/components/Loans/OpenLoans/helpers/isOverridePossible/overridePossibleMessages.js#L23
In case no changes are done on UI. this functionality will be broken.
The reason that messages are used here instead of something like error codes is that no approach was approved for these cases. As far as I know, code errors approach is used in some of the other modules.

Reply:
@dmytrotur @marcjohnson-kint could you please explain the process in such cases? Should I create PR to this module myself or create a task for someone from our team or another team?

Reply:
@bohdan-suprun I'm not sure about the general process. Seems like usually story is created for the team, that is responsible for this functionality. In this particular case, I can say that the Vega team has worked on this before. I would also prefer to wait for @marc-johnson to add his opinion.

Reply:
@dmytrotur, sounds good, thanks. 

@marcjohnson-kint are you ok with the above?

Reply:
If we are making changes that need synchronising across modules, then it is up to the developer who made the original changes to coordinate changes in other modules.

It is up to them as to whether they issue a pull request or ask others to do so (possibly via the scrum masters)

Reply:
Do we really need to change the error message?

In the scenario we are discussing, what is the profile of the policy itself?

Reply:
I am reluctant to start using codes until we have some kind of FOLIO wide agreement of how the are to be used. For example: How are they defined? Are they part of the interface or the implementation? How do we avoid clashes across modules?

Reply:
@bohdan-suprun sorry, what are you asking if I’m ok with?

Reply:
@marc-johnson Previously `RollingRenewalDueDateStrategy` was used only for loan policy with rolling profile, so `renewal date falls outside of date ranges in rolling loan policy` message was valid. But now, in some cases, this strategy is used in policy with a fixed profile, which makes this error message invalid.

Reply:
@marcjohnson-kint 

> Do we really need to change the error message?

If UI uses the error messages from response as is than it will at least confuse end-user in case he is renewing item and uses fixed policy profile but receives error that saying _'renewal date falls outside of date ranges in **rolling** loan policy'_.

> If we are making changes that need synchronising across modules, then it is up to the developer who made the original changes to coordinate changes in other modules.

Ok, so will create PR myself.

Reply:
@marcjohnson-kint, @dmytrotur PR has been created https://github.com/folio-org/ui-users/pull/954

Reply:
@marcjohnson-kint, @dmytrotur I've got approval on the UI PR, could you please approve this PR if you're ok?